databricks/koalas/tests/test_groupby.py

```diff
@@ -435,6 +435,42 @@ def test_describe(self):
             describe_pdf = pdf.groupby("a").describe().sort_index()
             self.assertRaises(NotImplementedError, lambda: kdf.groupby("a").describe().sort_index())
 
+    @unittest.skipIf(pd.__version__ < "1.0.0", "not supported before pandas 1.0.0")
```

Comment:
We can construct the output pandas dataframe and compare the results instead of skipping it.

Reply:
Okay It seems better. Thanks!